libs/SalesforceSwiftSDK/SalesforceSwiftSDK.xcodeproj/project.pbxproj

```diff
@@ -362,7 +362,8 @@
 				B7DA7AB61FCDE08E0098C1EA /* Sources */,
 				B7DA7AB71FCDE08E0098C1EA /* Frameworks */,
 				B7DA7AB81FCDE08E0098C1EA /* Headers */,
-				B7DA7AB91FCDE08E0098C1EA /* Resources */,
+				CE57D9001FD7B19700A0930F /* ShellScript */,
```

Comment:
Added 2 automated scripts - to generate umbrella header (for usage with `Obj-C`) and to generate pod specs when new files are added.

Reply:
Do we need the umbrella header since that library should only be used in swift apps?

Reply:
It can be used in obj-c too and this is the convention from what I've read online. But you're right - it's not a must have.

Reply:
Sounds good.